moto/eks/models.py

```diff
@@ -0,0 +1,467 @@
+CLUSTER_NOT_READY_MSG = BASE_MSG + "Cluster '{cluster_name}' is not in ACTIVE status"
+LAUNCH_TEMPLATE_WITH_DISK_SIZE_MSG = (
+    BASE_MSG + "Disk size must be specified within the launch template."
+)
+LAUNCH_TEMPLATE_WITH_REMOTE_ACCESS_MSG = (
+    BASE_MSG + "Remote access configuration cannot be specified with a launch template."
+)
+NODEGROUP_EXISTS_MSG = (
+    BASE_MSG
+    + "NodeGroup already exists with name {nodegroup_name} and cluster name {cluster_name}"
+)
+NODEGROUP_NOT_FOUND_MSG = BASE_MSG + "No node group found for name: {nodegroup_name}."
+
+
+class Cluster:
+    def __init__(
+        self,
+        name,
+        roleArn,
+        resourcesVpcConfig,
+        regionName,
+        awsPartition,
+        version=None,
+        kubernetesNetworkConfig=None,
+        logging=None,
+        clientRequestToken=None,
+        tags=None,
+        encryptionConfig=None,
+    ):
+        if encryptionConfig is None:
+            encryptionConfig = dict()
+        if tags is None:
+            tags = dict()
+
+        self.nodegroups = dict()
+        self.nodegroup_count = 0
+
+        self.arn = CLUSTER_ARN_TEMPLATE.format(
+            partition=awsPartition, region=regionName, name=name
+        )
+        self.certificateAuthority = {"data": random_string(1400)}
+        self.creation_date = iso_8601_datetime_without_milliseconds(datetime.now())
+        self.identity = {"oidc": {"issuer": ISSUER_TEMPLATE.format(region=regionName)}}
+        self.endpoint = ENDPOINT_TEMPLATE.format(region=regionName)
+
+        self.kubernetes_network_config = (
+            kubernetesNetworkConfig or DEFAULT_KUBERNETES_NETWORK_CONFIG
+        )
+        self.logging = logging or DEFAULT_LOGGING
+        self.platformVersion = DEFAULT_PLATFORM_VERSION
+        self.status = DEFAULT_STATUS
+        self.version = version or DEFAULT_KUBERNETES_VERSION
+
+        self.client_request_token = clientRequestToken
+        self.encryption_config = encryptionConfig
+        self.name = name
+        self.resources_vpc_config = resourcesVpcConfig
+        self.role_arn = roleArn
+        self.tags = tags
+
+    def __iter__(self):
+        yield "name", self.name
+        yield "arn", self.arn
+        yield "createdAt", self.creation_date
+        yield "version", self.version
+        yield "endpoint", self.endpoint
+        yield "roleArn", self.role_arn
+        yield "resourcesVpcConfig", self.resources_vpc_config
+        yield "kubernetesNetworkConfig", self.kubernetes_network_config
+        yield "logging", self.logging
+        yield "identity", self.identity
+        yield "status", self.status
+        yield "certificateAuthority", self.certificateAuthority
+        yield "clientRequestToken", self.client_request_token
+        yield "platformVersion", self.platformVersion
+        yield "tags", self.tags
+        yield "encryptionConfig", self.encryption_config
+
+    def isActive(self):
```

Comment:
Same here

Reply:
These need to stay camelCase, that is what the API accepts.  I had left the above two camelCase to align with that, but have changed them now.